Supremicus/DawnTitleCard.py

```diff
@@ -0,0 +1,545 @@
+                identifier='crt_state_overlay',
+                description='Enable CRT TV Watched/Unwatched overlay.',
+                tooltip=(
+                    'Enable CRT TV Watched/Unwatched overlay: <v>True</v> / <v>False</v>. Default is <v>False</v>.<br>'
+                    'Will only work if CRT TV Overlay is enabled.'
+                ),
+            ),
+            Extra(
+                name='Gradient Omission',
+                identifier='omit_gradient',
+                description='Whether to omit the gradient overlay',
+                tooltip=(
+                    'Either <v>True</v> or <v>False</v>. Set to <v>False</v> if you have '
+                    'trouble reading text on brighter images.<br>Default is <v>True</v>.'
+                ),
+            ),
+        ],
+        description=[
+            "Produce TitleCards with left, centered or right aligned text on the bottom.",
+            "Optional CRT TV overlay for shows like Stranger Things with a retro theme.",
+        ]
+    )
+
+    class CardModel(BaseCardTypeCustomFontAllText):
+        episode_text_vertical_shift: int = 0
+        title_text_horizontal_shift: int = 0
+        stroke_color: str = 'black'
+        separator: str = '•'
+        h_align: Literal['left', 'center', 'right'] = 'left'
+        symbol: str = None
+        crt_overlay: str = None
+        crt_state_overlay: bool = False
+        omit_gradient: bool = True
+
+    """Characteristics for title splitting by this class"""
+    TITLE_CHARACTERISTICS: SplitCharacteristics = {
+        'max_line_width': 28,
+        'max_line_count': 4,
+        'style': 'bottom',
+    }
+
+    """Characteristics of the default title font"""
+    TITLE_FONT = str(RemoteFile('Supremicus', 'ref/fonts/HelveticaNeue-Bold.ttf'))
+    TITLE_COLOR = 'white'
+    FONT_REPLACEMENTS = {}
+
+    """Whether this CardType uses season titles for archival purposes"""
+    USES_SEASON_TITLE = True
+
+    """Standard class has standard archive name"""
+    ARCHIVE_NAME = 'dawn'
+
+    """Characteristics of episode text"""
+    EPISODE_TEXT_FORMAT = 'EPISODE {to_cardinal(episode_number)}'
+    EPISODE_TEXT_FONT = str(RemoteFile('Supremicus', 'ref/fonts/ExoSoft-Medium.ttf'))
+
+    """Source path for CRT overlays to be overlayed if enabled"""
+    __OVERLAY_PLAIN = str(RemoteFile('Supremicus', 'ref/overlays/overlay_plain.png'))
+    __OVERLAY_PLAIN_BEZEL = str(RemoteFile('Supremicus', 'ref/overlays/overlay_plain_bezel.png'))
+    __OVERLAY_PLAY = str(RemoteFile('Supremicus', 'ref/overlays/overlay_play.png'))
+    __OVERLAY_PLAY_BEZEL = str(RemoteFile('Supremicus', 'ref/overlays/overlay_play_bezel.png'))
+    __OVERLAY_REWIND = str(RemoteFile('Supremicus', 'ref/overlays/overlay_rewind.png'))
+    __OVERLAY_REWIND_BEZEL = str(RemoteFile('Supremicus', 'ref/overlays/overlay_rewind_bezel.png'))
+
+    """Source path for the gradient image"""
+    __GRADIENT_IMAGE = str(RemoteFile('Supremicus', 'ref/overlays/gradient.png'))
```

Comment:
```suggestion
    EPISODE_TEXT_FONT = RemoteFile('Supremicus', 'ref/fonts/ExoSoft-Medium.ttf')

    """Source path for CRT overlays to be overlayed if enabled"""
    __OVERLAY_PLAIN = RemoteFile('Supremicus', 'ref/overlays/overlay_plain.png')
    __OVERLAY_PLAIN_BEZEL = RemoteFile('Supremicus', 'ref/overlays/overlay_plain_bezel.png')
    __OVERLAY_PLAY = RemoteFile('Supremicus', 'ref/overlays/overlay_play.png')
    __OVERLAY_PLAY_BEZEL = RemoteFile('Supremicus', 'ref/overlays/overlay_play_bezel.png')
    __OVERLAY_REWIND = RemoteFile('Supremicus', 'ref/overlays/overlay_rewind.png')
    __OVERLAY_REWIND_BEZEL = RemoteFile('Supremicus', 'ref/overlays/overlay_rewind_bezel.png')

    """Source path for the gradient image"""
    __GRADIENT_IMAGE = RemoteFile('Supremicus', 'ref/overlays/gradient.png')
```
This is so you can treat these as `Path` objects and call `resolve()` on them